configuration/exodus-lambda-deploy.yaml

```diff
@@ -30,10 +30,8 @@ Parameters:
     Description: A list of secret key IDs for Distribution key groups
 
 Conditions:
-  EnableKeyGroup: !Not
-    - !Equals
-      - !Join ["", !Ref keyids]
-      - ""
+  EnableKeyGroup:
+    !Not [!Equals [!Join ["", !Ref keyids], None]]
```

Comment:
This probably works but maybe the default value should be an empty list?  Not sure how that looks for CommaDelimitedList, perhaps `""`.  For a list of key IDs, "empty list" seems a better default value than either a string "None" or an actual `None`/`null`.

Reply:
Yeah, any string will work but None is already used in the template for default values so I stuck with that.

It doesn't really make sense as a default for any of the other parameters either. Maybe we should follow up with a PR to update them all.